src/main/java/com/example/demo/Application.java

```diff
@@ -0,0 +1,23 @@
+package com.example.demo;
+
+import com.example.demo.service.MainScreen;
+import lombok.AllArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
FYI:
org.springframework.boot.autoconfigure.SpringBootApplication#proxyBeanMethods
should be `false` to improve performance (in this particular case).